bitnami/zookeeper/values.yaml

```diff
@@ -209,6 +209,11 @@ nodeSelector: {}
 ##
 tolerations: []
 
+## Annotations
+## Ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+annotations: {}
```

Comment:
Small request, could we use "podAnnotations" instead of "annotations"? We're trying to standardize all the charts with this name.

Reply:
Hi, @juan131 
Done